Plugins/AutocompleteManager/AutocompleteManager.swift

```diff
@@ -106,7 +106,7 @@ open class AutocompleteManager: NSObject, InputPlugin, UITextViewDelegate, UITab
     
     // MARK: - Initialization
     
-    public init(for textView: InputTextView) {
+    public init(for textView: UITextView) {
         super.init()
```

Comment:
These are fine because no one is using the library but please don't make merging early a habit. This would be an API breaking change if the project was in use 😓